docs/units/index.rst

```diff
@@ -82,6 +82,13 @@ Units that "cancel out" become a special unit called the
 
     >>> u.m / u.m
     Unit(dimensionless)
+    
+To create a simple :ref:`dimensionless quantity <doc_dimensionless_unit>`, multiply 
+or divide a value by the unscaled dimensionless unit::
```

Comment:
I'd delete "or divide" here - it is true strictly speaking but more likely to confuse than to help (really, nobody should do `1 / u.dimensionless_unscaled`!

Reply:
Implemented!